packages/gestalt/src/FormElement.css

```diff
@@ -4,7 +4,7 @@
 
   /* hover colors */
   --gestalt-colorGray150Hovered: #d0d0d0;
-  --gestalt-colorRed100Hovered: #d80021;
+  --gestalt-colorRed100Hovered: #ad081b;
```

Comment:
This one was already in here with the new name. I went with the same red for hover both here and the button for consistency but could break it out again if this won't work.